src/components/InputWithLabel.jsx

```diff
@@ -0,0 +1,30 @@
+import "./inputWithLabel.css"
+function InputWithLabel(props) {
+    const type = props.Type;
+    if (type == "input") {
```

Comment:
using the string `input` as a test variable is more error-prone, so we can use a boolean variable for example `multiline` if it is true we return the textarea and else we return input.